src/main/java/jp/co/soramitsu/sora/didresolver/commons/DIDTypeEnum.java

```diff
@@ -6,8 +6,7 @@
 import lombok.val;
 
 public enum DIDTypeEnum {
-  IROHA("(?<ACCOUNT>\\w+)@(?<DOMAIN>\\w+[\\w|.]*)"),
-  UUID("[0-9a-f]{8}-[0-9a-f]{4}-[1-5][0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12}");
+  USERNAME("[a-zA-Z0-9]{6,32}");
```

Comment:
DID spec doesn't have such constraint:
```
idstring           = 1*idchar
idchar             = ALPHA / DIGIT / "." / "-"
```
should we comply with spec in this case, @Warchant ?

Reply:
if we have any constraints on username in account service, then they need to be implemented here also